cli/app.go

```diff
@@ -137,7 +138,7 @@ func main() {
 	// Same rule set applies to both TerraformBuiltInRules and Terraform11BuiltInRules
 	// Run for terraform12 by default
 	if useTerraformBuiltInRules {
-		builtInRuleSet, err := loadBuiltInRuleSet("terraform.yml")
+		builtInRuleSet, err := loadBuiltInRuleSet("terraform")
```

Comment:
I take it that the `cli/assets/terraform` directory is recursively used here instead of a single file. Can you update the code comments just above this to reflect that?

Reply:
updated